client/signup/steps/import-from/wordpress/import-everything/confirm.tsx

```diff
@@ -97,9 +105,18 @@ export const Confirm: React.FunctionComponent< Props > = ( props ) => {
 					) }
 				</SubTitle>
 
-				<NextButton onClick={ () => setIsModalDetailsOpen( true ) }>
-					{ __( 'Start import' ) }
-				</NextButton>
+				{ ! isTargetSitePlanCompatible && (
+					<>
+						<Notice>{ __( 'You need to upgrade your account to import everything.' ) }</Notice>
+						<NextButton onClick={ startImport }>{ __( 'See plans' ) }</NextButton>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'See Plans', { context: 'logged-out-homepage'} )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/onboarding-wordpress-importer-4)

Reply:
I think that we can put a capital **P** letter here without problems.

Reply:
Agree, but there is another button "Start import".
Should we put a capital "I" as well, I'm trying to be consistent?

Reply:
I've seen that elsewhere you can find buttons with all capitalized words and buttons without. So IMHO we can move as the Figma.